frontend/src/ducks/ledgers.ts

```diff
@@ -141,6 +143,32 @@ export const fetchLedgersTransactionsHistoryAction = createAsyncThunk<
   },
 );
 
+export const fetchLedgerOperations = createAsyncThunk<
+  FetchLedgerOperationsResponse[],
+  void,
```

Comment:
Should we use `undefined` instead of `void` here for the accepted parameter (or not param in this case)? `void` feels more like a type for a function that doesn't return anything. I'm not very opinionated about it, just curious what you think. :slightly_smiling_face:

Reply:
I prefer use `undefined`, but I left `void` following the design pattern, in some other functions they were already like this.

Reply:
Let's use `undefined`, we prefer it as well. We must have missed `void` :sweat_smile: . Thank you!